lua/neogit/lib/hl.lua

```diff
@@ -115,6 +115,7 @@ function M.setup()
     NeogitGraphBlue = { fg = palette.blue },
     NeogitGraphPurple = { fg = palette.purple },
     NeogitGraphGray = { fg = palette.grey },
+    NeogitGraphOrange = { fg = palette.orange },
```

Comment:
Can you add this to the documentation section for highlight groups? (`doc/neogit.txt:265`)

Reply:
Added along with `NeogitGraphGray` because it was missing. The docs also state `NeogitGraphBlack` but I think it was removed in a previous commit?

Reply:
Yea, I got my wires crossed between writing the highlighting logic and documenting it. Thanks for catching that :)